### PR TITLE
Feat/jm 7050

### PR DESCRIPTION
**Pull request by gregs-cgi**
### JIRA link (if applicable) ###

https://centralgovernmentcgi.atlassian.net/browse/JM-7050

### Change description ###

- [x] Juror record payments & attendance tab
- [x] Juror record history tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: client/templates/juror-management/juror-record/juror-history.njk [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% endblock %}
+
+{% block content %}
+  {% set currentName = [juror.commonDetails.title, juror.commonDetails.firstName, juror.commonDetails.lastName] %}
+
+  {% include "../_partials/heading.njk" %}
+
+  <h2>
+    History
+  </h2>
```
Comment (ricardofreitasrocha) [inline client/templates/juror-management/juror-record/juror-history.njk:23]: probably needs a `govuk-caption-(m|l)` class

File: server/routes/juror-management/juror-record/juror-record.controller.js [hunk 2]
```diff
@@ -1036,6 +1037,41 @@
...
+        historyTab,
+        history,
+        currentTab: 'history',
+        backLinkUrl: {
+          url: app.namedRoutes.build('juror-record.overview.get', { jurorNumber }),
+          built: true,
+        },
+      });
+    } catch (err) {
+      console.log(err);
```
Comment (ricardofreitasrocha) [inline server/routes/juror-management/juror-record/juror-record.controller.js:1070]: replace this with a logger call 😄

File: server/routes/juror-management/juror-record/juror-record.controller.js [hunk 3]
```diff
@@ -1036,6 +1037,41 @@
...
+      const jurorNumber = req.params.jurorNumber;
+      const {data: juror} = await jurorRecordObject.record.get(
+        require('request-promise'),
+        app,
+        req.session.authToken,
+        'overview',
+        jurorNumber,
+        req.session.locCode,
+      );
+
```
Comment (ricardofreitasrocha) [inline server/routes/juror-management/juror-record/juror-record.controller.js:1056]: we should probably have some logging on these calls if they succeed

**Review by ricardofreitasrocha — CHANGES_REQUESTED** [review]

File: server/routes/juror-management/juror-record/juror-record.controller.js (hunk 3 above)
Comment (gregs-cgi) [reply server/routes/juror-management/juror-record/juror-record.controller.js:1056]: Added

File: server/routes/juror-management/juror-record/juror-record.controller.js (hunk 2 above)
Comment (gregs-cgi) [reply server/routes/juror-management/juror-record/juror-record.controller.js]: Fixed

File: client/templates/juror-management/juror-record/juror-history.njk (hunk 1 above)
Comment (gregs-cgi) [reply client/templates/juror-management/juror-record/juror-history.njk]: Fixed - used the govuk-heading-l to match the other juror record tabs

**Review by ricardofreitasrocha — APPROVED** [review]

**Pull request merged**